lms/djangoapps/shoppingcart/models.py

```diff
@@ -980,6 +981,17 @@ class InvoiceTransaction(TimeStampedModel):
     created_by = models.ForeignKey(User)
     last_modified_by = models.ForeignKey(User, related_name='last_modified_by_user')
 
+    @classmethod
+    def get_invoice_transaction(cls, invoice_id):
```

Comment:
we have tried to using model.managers. We have created a separate PR for managers and going to be merged in this branch once test passes.
here is the PR link https://github.com/edx/edx-platform/pull/8101